README.md

```diff
@@ -1 +1,46 @@
-# dat
+# dat: Delta Acceptance Testing
+
+Warning: this project is incubating 🚧 🛠.  Any changes are possible in the future, including full deletion of this repo.
+
+This repo contains Delta Lake reference tables.  It specifies functionality that the [Delta connectors](https://github.com/delta-io/connectors) should be able to implement.  The Delta Lake connectors will use these reference tables in integration tests to demonstrate they can perform the requisite read / write operations on Delta Lakes.
+
+These tests will ensure that the various Delta connectors are interoperable and help deliver a wonderful user experience to end users.
+
+## Reference tables
+
+See the `reference_tables` directory for the tables you can access in this project.  Here's the layout of the reference tables directory:
+
+```
+reference_tables
+    reference_table1
+        table
+            _delta_log
+                N.checkpoint
+                N+1.json
+            uuid1.parquet
+            uuid2.parquet
+        table_metadata.json
+        table_content.parquet
+    reference_table2
+        ...
+    reference_table3
+        ...
+```
+
+Here's what each reference table contains:
+
+* `table`: A Delta Lake
+* `table_metadata.json`: metadata about the Delta Lake stored in a JSON file
+* `table_content.parquet`: the "final state" data that should be contained in the Delta Lake
+
+## Setup
```

Comment:
What/why do I need to setup anything? What if I have a connector in `go`? This setup won't apply to that.